src/pages/Pool/PositionPage.tsx

```diff
@@ -654,7 +661,12 @@ export function PositionPage() {
                     marginRight: '12px',
                   }}
                 >
-                  <div style={{ marginRight: 12 }}>
```

Comment:
I removed this because it's not currently centered from what I can tell:

<img width="449" alt="" src="https://user-images.githubusercontent.com/1091472/181688501-ad346075-b9df-496d-8297-4eda3323cc13.png">